### ISSUE TITLE
Tester Report - Papyrus errors and broken blackjack.

### ISSUE THREAD

**Issue by Scrivener07** [Scrivener07/FO4_Games#37]
Tested by **E**

> And now I just got busted out without my card being shown to me, I have 5, 4, 3 on the table, drew another card, told I was busted but no fourth card is shown
> There is a log of it...
> https://pastebin.com/4V9MeVPj
> ^ truncated
> Now after that hand it's not dealing cards
> Log section when dealing next hand after aboves missing fourth card:
> https://pastebin.com/dTfhaGc9
> (No cards actually dealt, no player controls, no exit from blackjack, had to exit the game)

**Comment by TheWhiteCollarPlayers** [comment]
We need more testing and refining for sure, if I can help with working this out let me know, my testing computer won't be ready for a few days.

**Comment by TheWhiteCollarPlayers** [comment]
from our voice discussion: we need more testing once UI stuff is done

**Comment by TheWhiteCollarPlayers** [comment]
Is this still a problem? Or do we need more testing on this?

**Comment by TheWhiteCollarPlayers** [comment]
Maybe this is related to the stack issue? with the game stopping I mean

**Comment by Scrivener07** [comment]
There is a good chance this was related to issue #48

**Comment by Scrivener07** [comment]
This is solved on master.
